python/ray/tune/suggest/optuna.py

```diff
@@ -329,22 +309,7 @@ def suggest(self, trial_id: str) -> Optional[Dict]:
                     cls=self.__class__.__name__,
                     metric=self._metric,
                     mode=self._mode))
-
-        if isinstance(self._space, list):
-            # Keep for backwards compatibility
-            # Deprecate: 1.5
```

Comment:
I think marking "deprecate: 1.X" is a good practice. We should adopt that rigorously as part of depreciation cycle.